js/bank.js

```diff
@@ -4,118 +4,113 @@ var rl = readline.createInterface({
   output: process.stdout
 });
 
-function start(balance) {
+
+function bank(balance) {
   this.balance = balance;
   var name = rl.question("Hello! Welcome to Citibank! What is you name? ", function(name) {
-    var decision = rl.question(name + ", would you like to: \n open an account, \n see your balance, \n make a withdrawal, \n make a deposit or \n close an account? ", function(decision){
-      if (decision == "balance") {
-        console.log("Your balance is $" + this.balance);
-        console.log("Goodbye, thanks for banking with Citibank!");
-        rl.close();
-      } else if (decision == "withdraw") {
-        this.balance = withdraw(this.balance);
-      } else if (decision == "deposit") {
-        this.balance = deposit(this.balance);
-      } else if (decision == "open an account") {
-        this.balance = open(this.balance);
-      } else if (decision == "close an account") {
-        this.balance = close(this.balance);
-      }
-      else {
-        var restart = rl.question("You did not make an appropriate decision, start from the beginning? Enter in yes, or no . " , function(restart) {
-          if (restart == "yes") {
-            start(this.balance);
-          }
-          else {
-            console.log("Goodbye, thanks for banking with Citibank!");
-             rl.close();
-            }
-        });
-      }
-    });
+    transaction(name, balance);
+  });
+}
+
+function transaction(name, balance) {
+  var transaction = rl.question(name + ", would you like to: \n"
+    + "1. See your balance \n"
+    + "2. Open an account \n"
+    + "3. Make a withdrawal \n"
+    + "4. Make a deposit \n"
+    + "5. Close an account \n"
+    + "Please type in the number corresponding to the option. ", function(transaction){
+    if (transaction == "1") {
+      console.log("Your balance is $" + this.balance);
+      rl.close();
+      message();
+    } else if (transaction == "2") {
+      this.balance = open(this.balance);
+    } else if (transaction == "3") {
+      this.balance = withdraw(this.balance);
+    } else if (transaction == "4") {
+      this.balance = deposit(this.balance);
+    } else if (transaction == "5") {
+      this.balance = close(this.balance);
+    } else {
+      restart();
+    }
   });
 }
 
+
 function open(balance) {
-  // need to add input here
+  this.balance = balance;
   console.log("Your Account has been created and the balance you have is $" + this.balance + ".");
-  console.log("Goodbye, thanks for banking with Citibank!");
   rl.close();
+  message();
 }
 
 function close(balance) {
```

Comment:
How about `closeAccountMessage`

Reply:
I think it will be really confusing to call it closeAccountMessage...maybe just closeAccount.